Source/WTF/wtf/PlatformUse.h

```diff
@@ -279,6 +279,7 @@
 
 #if PLATFORM(IOS) || PLATFORM(MACCATALYST)
 #define USE_UICONTEXTMENU 1
+#define USE_UIKIT_CONTEXT_MENU_INTERACTION (defined(UIKIT_SUPPORTS_CONTEXT_MENU_INTERACTION) && UIKIT_SUPPORTS_CONTEXT_MENU_INTERACTION)
```

Comment:
I think you can also just use the existing HAVE_TEXT_INTERACTION_WITH_CONTEXT_MENU_INTERACTION flag as well, instead of adding a new one?

Reply:
oh that's perfect, I'll use that instead, thank you